geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/data/KeyHashUtil.java

```diff
@@ -35,7 +33,7 @@ public static short slotForKey(byte[] key) {
     int endHashtag;
 
     for (startHashtag = 0; startHashtag < key.length; startHashtag++) {
-      if (key[startHashtag] == LEFT_BRACE_ID) {
+      if (key[startHashtag] == (byte) '{') {
```

Comment:
I think all these casts of "(byte)" can be removed.
Given a variable of type "byte" named "b" you can do "b == '{'". I think removing these will make the code a bit more readable.